app/controllers/v0/sessions_controller.rb

```diff
@@ -112,13 +104,32 @@ def set_cookies
       set_sso_cookie! # Sets a cookie "vagov_session_<env>" with attributes needed for SSO.
     end
 
+    def new_query_params(type)
+      query_params = {}
+      query_params[:clientId] = params[:clientId] if params[:clientId]
+      if %w[mhv dslogon idme].include?(type)
```

Comment:
Maybe track MFA and identity proofing too if we'd like to report analytics for successes for those flows.